README.md

```diff
@@ -35,6 +35,12 @@ yarn add typed.js
 bower install typed.js
 ~~~
 
+#### CDN
+
+~~~
+<script href=" https://cdn.jsdelivr.net/npm/typed.js@2"></script>
```

Comment:
@LukasDrgon `href` should have been `src`

Reply:
![](https://i.imgur.com/iWKad22.jpg)